torcheeg/trainers/classifier.py

```diff
@@ -53,41 +53,58 @@ class ClassifierTrainer(pl.LightningModule):
             trainer.fit(train_loader, val_loader)
             trainer.test(test_loader)
 
+           
+        
         Args:
-            model (nn.Module): The classification model, and the dimension of its output should be equal to the number of categories in the dataset. The output layer does not need to have a softmax activation function.
+            model (nn.Module): The classification model,make sure its ouput dimension align to num_class.
             num_classes (int, optional): The number of categories in the dataset. If :obj:`None`, the number of categories will be inferred from the attribute :obj:`num_classes` of the model. (defualt: :obj:`None`)
```

Comment:
Make `num_classes` a required parameter and remove the need to infer it from the model or data. Please delete `__try_init_num_classes` and `get_MissingParameter`.